src/parser/data/Variable.h

```diff
@@ -9,6 +9,7 @@
 
 #include "../../engine/ResultTable.h"
 #include "../../index/Index.h"
+#include "Context.h"

```

Comment:
`Context.h` seems to provide some sort of context for the evaluation of a Variable. So that evaluation function knows which type the variable is and can lookup the value etc.I didn't change anything here directly but had to add it so that it would compile. Probably changed something upstream that caused this to be needed but I'm not sure.